sections/errorhandling/usematurelogger.md

```diff
@@ -2,43 +2,19 @@
 
 ### One Paragraph Explainer
 
-We all love console.log but obviously, a reputable and persistent logger like [Winston][winston] (highly popular) or [Pino][pino] (the new kid in town which is focused on performance) is mandatory for serious projects. A set of practices and tools will help to reason about errors much quicker – (1) log frequently using different levels (debug, info, error), (2) when logging, provide contextual information as JSON objects, see example below. (3) Watch and filter logs using a log querying API (built-in in most loggers) or a log viewer software. (4) Expose and curate log statement for the operation team using operational intelligence tools like Splunk.
+We all love console.log but obviously, a reputable and persistent logger like [Pino][pino] (a [newer option focused on performance][dropdiscussion]) is mandatory for serious projects. 
 
-[winston]: https://www.npmjs.com/package/winston
-[pino]: https://www.npmjs.com/package/pino
+A set of practices and tools will help to reason about errors much quicker.
+Recommendations include:
+
+1. Log frequently using different levels (debug, info, error).

```

Comment:
We try to keep the One paragraph Explainer a paragraph as much as possible :) Could you remove the line-breaks on line 6 and 7 at least?

Reply:
Good idea. Please see updated version.